modules/Bio/EnsEMBL/DBSQL/BiotypeAdaptor.pm

```diff
@@ -194,4 +194,39 @@ sub fetch_all_by_object_type {
   return \@biotypes;
 }
 
+=head2 fetch_all_by_group_object_db_type
+
+  Arg [1]    : String $biotype_group
+               The group of the biotypes to retrieve
+  Arg [2]    : String $object_type
+               The object type of the biotypes to retrieve (gene or transcript)
+  Arg [3]    : (optional) String $db_type
+               The db_type of the biotypes to retrieve. If not provided defaults to 'core'.
+  Example    : $biotype = $biotype_adaptor->fetch_all_by_group_object_db_type('coding', 'gene');
+  Description: Retrieves an array reference of biotype objects from the database of the provided
+               biotype_group and object_type and core db_type.
+  Returntype : arrayref of Bio::EnsEMBL::Biotype objects or empty arrayref
+  Warning    : If empty arrayref is to be returned
+  Exceptions : none
+
+=cut
+
+sub fetch_all_by_group_object_db_type {
+  my ($self, $biotype_group, $object_type, $db_type) = @_;
+
+  $db_type //= 'core';
+
+  my $constraint = "b.biotype_group = ? AND b.object_type = ? AND FIND_IN_SET( ? , db_type)>0 ";
```

Comment:
This one isn't going to work with SQLite.

Reply:
usually test with mysql so missed that...
Have updated the query and should now work in both, though not as elegant.